sources/functions/app_port

```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/env bash
+#
+declare -A app_port
+#
+# List our ports here using the forman array_name[app_name]port
+app_port[authelia]=9091
+app_port[filebrowser]=9090
+app_port[jackett]=9117
```

Comment:
I swear we had at least two more apps hmmmm

Reply:
I think where the array is stored matters more. `/swizzin/ports` or something. App only need ot added as the scripts are updated.